week-2-wide-perspective/w02e05-simple-server-side/prisma/schema.prisma

```diff
@@ -0,0 +1,55 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Product {
+  id    String     @id @default(uuid())
+  name  String
+  price Float
+
+  orders OrdersOnProducts[]
+}
+
+enum Role {
+    USER
+    ADMIN
+}
+
+model User {
+    login String @id @unique
+    password String
+
+    orders Order[]
+    name String
+    role Role
+}
+
+model Order {
+  id    String     @id @default(uuid())
+  userLogin  String
+  user User @relation(fields: [userLogin], references: [login], onDelete: Cascade)
+
+  products OrdersOnProducts[]
+  createdAt DateTime @default(now())
+}
+
+model OrdersOnProducts {
```

Comment:
Generalnie tutaj jest bardzo fajnie. Brakuje `Invoice` ale to nie ma problemu - chodziło właśnie o to żebyś sobie na spokojnie przeszedł przez wszystkie elementy i warstwy - więc jest git.